### ISSUE TITLE
Test failures against git-annex 8.20200908

### ISSUE THREAD

**Issue by risicle** [alpernebbi/git-annex-adapter#13]
Is `git-annex-adapter` intended to work against `git-annex` 8.x?

We get the following test failures

```
running install tests
.....F.F./nix/store/cpzs1hpwzs23c41haa4dap0zjfx6xych-python3-3.7.9/lib/python3.7/subprocess.py:883: ResourceWarning: subprocess 493 is still running
  ResourceWarning, source=self)
ResourceWarning: Enable tracemalloc to get the object allocation traceback
./nix/store/cpzs1hpwzs23c41haa4dap0zjfx6xych-python3-3.7.9/lib/python3.7/subprocess.py:883: ResourceWarning: subprocess 670 is still running
  ResourceWarning, source=self)
ResourceWarning: Enable tracemalloc to get the object allocation traceback
./nix/store/cpzs1hpwzs23c41haa4dap0zjfx6xych-python3-3.7.9/lib/python3.7/subprocess.py:883: ResourceWarning: subprocess 889 is still running
  ResourceWarning, source=self)
ResourceWarning: Enable tracemalloc to get the object allocation traceback
..F............/nix/store/cpzs1hpwzs23c41haa4dap0zjfx6xych-python3-3.7.9/lib/python3.7/subprocess.py:883: ResourceWarning: subprocess 1646 is still running
  ResourceWarning, source=self)
ResourceWarning: Enable tracemalloc to get the object allocation traceback
.
======================================================================
FAIL: test_init_annex_version_five (tests.test_init_annex.TestInitAnnexOnEmptyRepo)
Repository version 5 should be valid.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/source/tests/test_init_annex.py", line 58, in test_init_annex_version_five
    self.assertEqual(annex_repo.config['annex.version'], '5')
AssertionError: '8' != '5'
- 8
+ 5


======================================================================
FAIL: test_init_annex_version_six (tests.test_init_annex.TestInitAnnexOnEmptyRepo)
Repository version 7 should be valid.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/source/tests/test_init_annex.py", line 63, in test_init_annex_version_six
    self.assertEqual(annex_repo.config['annex.version'], '7')
AssertionError: '8' != '7'
- 8
+ 7

======================================================================
FAIL: test_process_annex_info_batch (tests.test_process.TestProcessOnEmptyAnnex)
Process should be able to read multiple lines
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/source/tests/test_process.py", line 229, in test_process_annex_info_batch
    'remote annex size: 0 bytes',
AssertionError: Lists differ: ['trust: semitrusted', 'remote annex keys: 0'] != ['remote annex keys: 0', 'remote annex size: 0 bytes']

First differing element 0:
'trust: semitrusted'
'remote annex keys: 0'

- ['trust: semitrusted', 'remote annex keys: 0']
+ ['remote annex keys: 0', 'remote annex size: 0 bytes']

----------------------------------------------------------------------
Ran 27 tests in 8.325s

FAILED (failures=3)
```

which worked with earlier versions of `git-annex`.

**Comment by alpernebbi** [comment]
These should be fixed by [`4257b9d`](https://github.com/alpernebbi/git-annex-adapter/commit/4257b9d0ca9176050cc5bf5fd668c52606cfc011) ("Correctly test initializing repo versions 5 and 6") and [`1c5a2165`](https://github.com/alpernebbi/git-annex-adapter/commit/1c5a21659f69c88e0ea28b62fbd30fb6f5ba37b0) ("Skip broken annex info batch process test"). They're already in the v0.2.2 release, but AFAIK NixOS hasn't upgraded to that version yet.

**Comment by risicle** [comment]
Ah - I'm an idiot - I thought I was using 0.2.2 but I was using 0.2.1 all along... will bump it ASAP.
